core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -74,9 +74,9 @@ struct GraphImpl<Kokkos::Cuda> {
   using root_node_impl_t =
       GraphNodeImpl<Kokkos::Cuda, Kokkos::Experimental::TypeErasedTag,
                     Kokkos::Experimental::TypeErasedTag>;
-  using aggregate_kernel_impl_t = CudaGraphNodeAggregateKernel;
+  using aggregate_impl_t = CudaGraphNodeAggregate;
```

Comment:
I find it somewhat confusing to have both `aggregate_impl_t` and `aggregate_node_impl_t`.
I would "inline" the definition and replace aggregate_impl_t -> CudaGraphNodeAggregate below.

Reply:
The thing is, having a `aggregate_impl_t` alias in `Kokkos::Impl::GraphImpl` is handy in the tests. Otherwise, I would need to guard with macros in the tests where I need the aggregate type. But agreed, it's confusing a bit.